### ISSUE TITLE
State change error for Ratgdo bridged through HA

### ISSUE THREAD

**Issue by mlamoure** [FlyingDiver/Indigo-HA-Agent#16]
Since moving my Ratgdo to the ESP firmware and then bridging from HA to Indigo via HA Agent, I get these messages when the door is in an opening or closing state.  I dont believe it has any actual impact on functionality, just spams the log.

```
Apr 19, 2024 at 12:54:44 PM
   Home Assistant Agent Error      Websocket state_changed exception: type object 'kStateImageSel' has no attribute 'Open'

```

**Comment by mlamoure** [comment]
Here is the Device Details Log:

```
   Script                          address : cover.ratgdov25i_e6fd15_door
batteryLevel : None
buttonConfiguredCount : 0
buttonGroupCount : 0
configured : True
description : 
deviceTypeId : ha_cover
displayStateId : onOffState
displayStateImageSel : Closed
displayStateValRaw : False
displayStateValUi : Closed
enabled : True
energyAccumBaseTime : None
energyAccumTimeDelta : None
energyAccumTotal : None
energyCurLevel : None
errorState : 
folderId : 0
globalProps : MetaProps : (dict)
     no.homeassistant.plugin : (dict)
          SupportsCloseTilt : false (bool)
          SupportsOpenTilt : false (bool)
          SupportsSetPosition : true (bool)
          SupportsSetTilt : false (bool)
          SupportsStatusRequest : false (bool)
          SupportsStop : true (bool)
          SupportsStopTilt : false (bool)
          address : cover.ratgdov25i_e6fd15_door (string)
          states_list : (list)
               Item : current_position (string)
               Item : device_class (string)
               Item : friendly_name (string)
               Item : supported_features (string)
id : 674191203
lastChanged : 2024-04-19 12:57:43
lastSuccessfulComm : 2024-04-19 12:57:43
ledStates : []
model : Home Assistant Cover
name : Garage Door (radgo via HA)
onState : False
ownerProps : no.homeassistant.plugin : (dict)
     SupportsCloseTilt : false (bool)
     SupportsOpenTilt : false (bool)
     SupportsSetPosition : true (bool)
     SupportsSetTilt : false (bool)
     SupportsStatusRequest : false (bool)
     SupportsStop : true (bool)
     SupportsStopTilt : false (bool)
     address : cover.ratgdov25i_e6fd15_door (string)
     states_list : (list)
          Item : current_position (string)
          Item : device_class (string)
          Item : friendly_name (string)
          Item : supported_features (string)
pluginId : no.homeassistant.plugin
pluginProps : emptyDict : (dict)
protocol : Plugin
remoteDisplay : True
sharedProps : com.indigodomo.indigoserver : (dict)
states : States : (dict)
     actual_state : closed (string)
     current_position : 0 (integer)
     device_class : garage (string)
     friendly_name : ratgdov2.5i e6fd15 Door (string)
     lastUpdated : 2024-04-19T16:57:43.502402+00:00 (string)
     onOffState : off (on/off bool)
     onOffState.ui : Closed (string)
     supported_features : 15 (integer)
subModel : 
subType : 
supportsAllLightsOnOff : False
supportsAllOff : False
supportsOnState : True
supportsStatusRequest : False
version : None

```

**Comment by mlamoure** [comment]
@FlyingDiver - tagging you directly because I believe your GitHub auto-assignment job is failing.

**Comment by FlyingDiver** [comment]
Edit like 674 of plugin.py from "Open" to "Opened" until I roll the next release.

**Comment by FlyingDiver** [comment]
And assign.yaml is in the workflows folder, not sure why it's not working.

**Comment by FlyingDiver** [comment]
Released in 2024.0.5
